Source/UI/Areas/Api/Controllers/PlayedGamesController.cs

```diff
@@ -122,7 +122,7 @@ internal virtual HttpResponseMessage GetPlayedGameSearchResults(PlayedGameFilter
         {
             var filter = _transformer.Transform<PlayedGameFilter>(playedGameFilterMessage);
 
-            var searchResults = _playedGameRetriever.SearchPlayedGames(filter);
+            var searchResults = _playedGameRetriever.SearchPlayedGames(filter, null);

```

Comment:
I think there is a CurrentUser on the base class so you should be able to just use that. The API controller base class (ApiControllerBase) automatically sets this for you. In regards to my other comment, I'm thinking we should allow a null user to be passed in so long as that IncludeCurrentPlayerGamesOnly isn't set to true. You essentially already have that, I'd just prefer to explicitly throw an ArgumentNullException in that case rather than letting it hit a NullReferenceException